src/components/screens/HistoryScreen/HistoryScreen.tsx

```diff
@@ -0,0 +1,229 @@
+/* eslint-disable react/no-unused-prop-types */
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable react/prop-types */
```

Comment:
friendly reminder to check if we could remove some of those with the current version of this file